src/server/redis_reply.h

```diff
@@ -32,10 +32,12 @@ namespace Redis {
 void Reply(evbuffer *output, const std::string &data);
 std::string SimpleString(const std::string &data);
 std::string Error(const std::string &err);
-std::string Integer(int64_t data);
+template <typename IntegerType>
+std::string Integer(IntegerType data);
```

Comment:
Actually you can implement this template function in the header directly to avoid explicit instantiations.